docs/stargz/stargz_guide.md

```diff
@@ -0,0 +1,30 @@
+# Running stargz-based containers with Knative
+
+This guide describes how to run stargz images on a single-cluster node using Knative's CLI.
```

Comment:
I would integrate this into the quick_start guide and make it a toggleable parameter as UPF (`snapshots` flags) is supported.

Reply:
Should this then be simply added as an extra step with a small description linking the `stargz` project or have a separate, smaller part that describes stargz as it is done now?

Reply:
Yes. I would go that way to avoid duplicating setup scripts and setup guides.